deploy/install-dev.sh

```
@@ -0,0 +1,238 @@
+fi
+
+
+
+# 询问是否开启鉴权(Y/N)
+read -p "是否开启鉴权(Y/N): " isAuth
+if [[ "$isAuth" =~ ^[Yy]$ ]]; then
+    read -p "请输入用户名: " username
+    read -p "请输入密码: " password
+    read -p "请输入密钥(默认rand): " secret
+    # 若secret不为空，则写入配置文件
+    cd ${bin_dir}
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)